Source/WTF/wtf/Vector.h

```diff
@@ -1121,15 +1121,19 @@ template<typename T, size_t inlineCapacity, typename OverflowHandler, size_t min
 template<typename U>
 bool Vector<T, inlineCapacity, OverflowHandler, minCapacity, Malloc>::contains(const U& value) const
 {
-    return find(value) != notFound;
+    for (const auto& elem : *this) {
```

Comment:
This is more verbose and the compiler is likely to generate the same code. Again, without proven perf benefits, why do this?

Reply:
Matthieu and I looked at this together and discovered that both versions of the generated loop produce the same assembly code using a clang release build on ARM64. (We didn't test Intel.)

Reply:
```
WTF::AutomaticThreadCondition::contains(WTF::AbstractLocker const&, WTF::AutomaticThread*):
0000000000000650        ldr     w9, [x0, #0x14]
0000000000000654        cbz     w9, 0x67c
0000000000000658        ldr     x8, [x0, #0x8]
000000000000065c        sub     x9, x9, #0x1
0000000000000660        ldr     x10, [x8], #0x8
0000000000000664        cmp     x10, x2
0000000000000668        cset    w0, eq
000000000000066c        ccmp    x9, #0x0, #0x4, ne
0000000000000670        sub     x9, x9, #0x1
0000000000000674        b.ne    0x660
0000000000000678        ret
000000000000067c        mov     w0, #0x0
0000000000000680        ret
```